core/src/test/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionFileSystemOutputTest.java

```diff
@@ -50,6 +53,9 @@ public TransmissionFileSystemOutputTest() {
         workingFolder = new File(LocalFileSystemUtils.getTempDir(), TEMP_TEST_FOLDER).getAbsolutePath();
```

Comment:
I think `workingFolder` is obsolete and can be removed. Right?

Reply:
Removed!